examples/istio_rate_limiting/README.md

```diff
@@ -0,0 +1,37 @@
+# Istio Wasm Demo for Rate Limiting
```

Comment:
we already push the examples' binary [(on main branch)](https://github.com/tetratelabs/proxy-wasm-go-sdk/blob/main/.github/workflows/push-wasm-images.yaml) to GHCR https://github.com/tetratelabs/proxy-wasm-go-sdk/pkgs/container/proxy-wasm-go-sdk-examples, so I think we should move this doc under /doc rather than under example/**/REAMDME.md?